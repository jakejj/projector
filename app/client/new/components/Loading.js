import React from 'react'
import { Grid, Row, Col } from 'react-bootstrap'
import ReactSpinner from 'react-spinjs';


export default (props) => {
  return (
    <Grid fluid={true} style={{marginTop: '5em'}}>
      <Row>
        <Col md={12}>
          <ReactSpinner config={{ lines: 13, length: 35, width: 22, radius: 61, scale: 0.25, corners: 0.8, color: '#000', opacity: 0.2, rotate: 0, direction: 1, speed: 0.9, trail: 50, fps: 20, zIndex: 2e9, className: 'spinner', top: '50%', left: '50%', shadow: false, hwaccel: false, position: 'absolute' }} />
        </Col>
      </Row>
    </Grid>
  )
}
