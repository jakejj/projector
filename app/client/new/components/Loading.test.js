import React from 'react'
import renderer from 'react-test-renderer';
import Loading from './Loading'

test('', () => {
  const component = renderer.create(
    <Loading />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

  // // manually trigger the callback
  // tree.props.onMouseEnter();
  // // re-rendering
  // tree = component.toJSON();
  // expect(tree).toMatchSnapshot();
  //
  // // manually trigger the callback
  // tree.props.onMouseLeave();
  // // re-rendering
  // tree = component.toJSON();
  // expect(tree).toMatchSnapshot();
});
