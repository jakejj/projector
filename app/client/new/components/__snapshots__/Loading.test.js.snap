exports[`test  1`] = `
<div
  className="container-fluid"
  style={
    Object {
      "marginTop": "5em",
    }
  }>
  <div
    className="row">
    <div
      className="col-md-12">
      <span />
    </div>
  </div>
</div>
`;
